spec/policies/accounting/transaction_policy_spec.rb

```diff
@@ -11,83 +11,91 @@
   let(:described_transaction) { Accounting::Transaction.new(project: loan) }
   subject(:policy) { described_class.new(user, described_transaction) }
 
-  shared_examples_for 'returns no reasons even if issues other than user role' do
+  shared_examples_for "returns no reasons even if issues other than user role" do
     it { expect(policy.read_only_reasons).to be_empty }
 
-    context 'with other issues present' do
+    context "with other issues present" do
       let(:loan_trait) { :completed }
       it { expect(policy.read_only_reasons).to be_empty }
     end
   end
 
-  context 'with non-admin' do
-    let(:user) { create(:user) }
-    forbid_all
-    it_behaves_like 'returns no reasons even if issues other than user role'
+  context "with non-admin" do
+    context "user has access to this division" do
+      let(:user) { create_member(division) }
```

Comment:
I see there are `:member` and `:admin` traits on the `User` factory. Is there a reason not to use those and perhaps phase out `create_member` and `create_admin`?